### PR TITLE
hdfs: add service check

### LINKED ISSUE #153: HDFS Service Check

### PR DESCRIPTION
**Pull request by kpgtek**
#### Which issue(s) this PR fixes:

Fixes #153

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by leopaul36 — APPROVED** [review]
Nice job @kpgtek!

**Review by mehdibn — APPROVED** [review]

File: playbooks/hdfs_check.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- name: HDFS service check
+  hosts: edge
+  environment:
+    LC_ALL: C
```
Comment (gboutry) [inline playbooks/hdfs_check.yml:8]: Do you tests depend on locale aware messages ?

Comment (kpgtek) [reply playbooks/hdfs_check.yml:8]: It seems that could ensure that most of common Linux commands output messages in english in the check.

**Review by rpignolet — APPROVED** [review]

**Comment by kpgtek** [comment]
During the previous merges and rebase, the following redundant port variables have been restored to ```hdfs.yml```:
```
# HDFS component listen port
hdfs_nn_rpc_port: 8020
hdfs_nn_http_port: 9870
hdfs_nn_https_port: 9871
hdfs_dn_rpc_port: 9866
hdfs_dn_https_port: 9865
hdfs_jn_rpc_port: 8485
hdfs_jn_https_port: 8481
```

**Pull request merged**